Funcky/Extensions/EnumerableExtensions/ForEach.cs

```diff
@@ -16,5 +17,13 @@ public static void ForEach<T>(this IEnumerable<T> elements, Action<T> action)
                 action(element);
             }
         }
+
+        /// <summary>
+        /// The IEnumerable version of foreach. You can apply an Func{T, Unit} to each element. This is only useful when you have side effects.
+        /// </summary>
+        /// <typeparam name="T">the inner type of the enumerable.</typeparam>
+        public static Unit ForEach<T>(this IEnumerable<T> elements, Func<T, Unit> action)
+            => elements
```

Comment:
You could also use `UnitToAction` here

Reply:
The ForEach extension returns Unit, therefore is the aggregate solution probably the best one.